rosidl_generator_py/resource/_msg.py.template

```diff
@@ -1,32 +1,24 @@
 # generated from rosidl_generator_py/resource/_msg.py.template
 # generated code does not contain a copyright notice
 
-class @(spec.base_type.type)__Constants(type):
-@[for constant in spec.constants]@
 
-    @@property
-    def @(constant.name)(cls):
-        """Message constant '@(constant.name)'"""
-        return @constant_value_to_py(constant.type, constant.value)
-@[end for]@
-@[for field in spec.fields]@
-@[if field.default_value]@
+class Metaclass(type):
+    """MetaClass of message '@(spec.base_type.type)'."""
```

Comment:
nitpik - Metaclass is a single word (computer science only) and so it doesn't need camel case here: https://en.wikipedia.org/wiki/Metaclass